factory/factory.go

```diff
@@ -34,6 +34,10 @@ func InitConfigFactory(f string) error {
 		if AmfConfig.Configuration.WebuiUri == "" {
 			AmfConfig.Configuration.WebuiUri = "webui:9876"
 		}
+		if AmfConfig.Configuration.KafkaInfo.EnableKafka == nil {
```

Comment:
Would not this mean that this will always be true (i.e., not `nil`) if the `enableKafka` field is present (regardless of its value)? or am I overlooking something?